src/commands/wr.js

```diff
@@ -0,0 +1,112 @@
+require('dotenv').config();
+const { SlashCommandBuilder } = require('@discordjs/builders');
+const { MessageEmbed } = require('discord.js');
+
+const discordPics = {
+    0: 'https://i.imgur.com/T9l6udJ.png',
+    1: 'https://i.imgur.com/iqbgcoo.png',
+    2: 'https://i.imgur.com/RAHIoie.png',
+    3: 'https://i.imgur.com/oPVaywI.png',
+    4: 'https://i.imgur.com/HHs9Pwf.png',
+    5: 'https://i.imgur.com/5U1Vrqb.png',
+    6: 'https://i.imgur.com/WNT8c0k.png',
+    7: 'https://i.imgur.com/e4PQWp8.png',
+    8: 'https://i.imgur.com/J7sNqiZ.png',
+    9: 'https://i.imgur.com/urD420H.png',
+    10: 'https://i.imgur.com/JQN0Mrz.png',
+};
+
+module.exports = {
+    data: new SlashCommandBuilder()
+        .setName('wr')
+        .setDescription(
+            'Retrieve the current world record for a specified map from the SourceJump database.',
+        )
+        .addStringOption((option) =>
+            option
+                .setName('map')
+                .setDescription('Name of the map to check')
+                .setRequired(true),
+        ),
+    async execute(interaction) {
+        // Maybe store this in env so it doesn't need to be changed/declared here?
+        const SOURCEJUMP_API_URL = 'https://sourcejump.net/api';
+        const fetch = require('node-fetch');
+        const map = interaction.options.getString('map');
+        const apiOptions = {
+            method: 'GET',
+            headers: {
+                'api-key': process.env.SJ_API_KEY,
+            },
+        };
+
+        fetch(`${SOURCEJUMP_API_URL}/records/${map}`, apiOptions)
+            .then((response) => response.text())
+            .then((body) => {
+                if (body.length === 2) {
```

Comment:
This would need to be changed to `body.length === 0` if `text()` is changed to `json()`.